### DISCUSSION TITLE
Obcommands

### DISCUSSION THREAD

**Discussion by jin-ahn** [esm7/obsidian-vimrc-support#83] [Q&A]
I am having trouble finding the obcommands. I press ctrl-shift-i like the README says and when I type `:obcommand` nothing shows up 
![image](https://user-images.githubusercontent.com/66384196/148673318-dc70881a-cf7f-4f32-83d9-48ab15b949f4.png)

**Comment by esm7** [comment]
Try to type `:obcommand` in an editor (as a Vim command), not the developer console.
